src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/Helper.java

```diff
@@ -245,19 +254,30 @@ private String function_getFullQualifiedName(String Name, PHPNamespaceDescriptor
         return "FUNCTION|" + fullname;
     }
     
+    /**
```

Comment:
Please remove the return value and provide some more information what is happening here.